doc/cascadia/Panes.md

```diff
@@ -0,0 +1,232 @@
+focused again.
+
+The Tab's state will be updated to reflect the state of it's focused pane. The
+title text and icon of the tab will reflect that of the focused pane. Should the
+focus switch from one pane to another, the tab's text and icon should update to
+reflect the newly focused control. Any additional state that the tab would
+display for a single pane should also be reflected in the tab for a tab with
+multiple panes.
+
+While panes are open, the user should be able to move any split between panes.
+In moving the split, the sizes of the terminal controls should be resized to
+match.
+
+### Closing a pane
+
+A pane can either be close by the user manually, or when the terminal it's
+attached to raises it's ConnectionClosed event. When this happens, we should
+remove this pane from the tree. The parent of the closing pane will have to
+remove the pane as one of it's children. If the sibling of the closing pane is a
+leaf, then the parent should just take all of the state from the remaining pane.
+This will cause the remaining pane's content to expand to take the entire
+boundaries of the parent's pane. If the remaining child was a parent itself,
+then the parent will take both the children of the remaining pane, and make them
+the parent's children, as if the parent node was taken from the tree and
+replaced by the remaining child.
+
+## Future considerations
+
+The Pane implementation isn't complete in it's current form. There are many
+additional things that could be done to improve the user experience. This is by
+no means a comprehensive list.
+
+* [ ] Panes should be resizable with the mouse. The user should be able to drag
+  the separator for a pair of panes, and have the content between them resize as
+  the separator moves.
+* [ ] There's no keyboard shortcut for "ClosePane"
+* [ ] The user should be able to configure what profile is used for splitting a
+  pane. Currently, the default profile is used, but it's possible a user might
+  want to create a new pane with the parent pane's profile.
+* [ ] There should be some sort of UI to indicate that a particular pane is
+  focused, more than just the blinking cursor. `tmux` accomplishes this by
+  colorizing the separators adjacent to the active pane. Another idea is
+  displaying a small outline around the focused pane (like when tabbing through
+  controls on a webpage).
+* [ ] The user should be able to navigate the focus of panes with the keyboard,
+  instead of requiring the mouse.
+* [ ] The user should be able to zoom a pane, to make the pane take the entire
+  size of the terminal window temporarily.
+* [ ] A pane doesn't necessarily need to host a terminal. It could potentially
+  host another UIElement. One could imagine enabling a user to quickly open up a
+  Browser pane to search for a particular string without needing to leave the
+  terminal.
+
+## Footnotes
+
+### Why not top-Level Panes, and nested tabs?

```

Comment:
>top-Level Panes, and nested tabs [](start = 12, length = 32)

I feel like this should be either "Top-Level Panes, and Nested Tabs" or "top-level panes, and nested tabs".